src/Flex.php

```diff
@@ -402,7 +403,8 @@ public function install(Event $event = null)
             copy($rootDir.'/.env.dist', $rootDir.'/.env');
         }
 
-        $recipes = $this->fetchRecipes();
+        $recipes = $this->fetchRecipes($this->operations);
+        $this->operations = [];     // Reset the operation after getting recipes
```

Comment:
Review note: This was done so that we could re-use `$this->fetchRecipes()` elsewhere - but keep "clearing" `$this->operations`, which the existing code does currently.